test-benchmark-visualizer.html

```diff
@@ -0,0 +1,136 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../paper-styles/color.html">
+<link rel="import" href="test-benchmark-visualizer/test-benchmark-sparkline.html">
+
+<!--
+This element is meant to be a light-weight visualization that ships with the
+`test-benchmark` element. It displays a series of cards, each of which
+correspond to a configured test in the `test-benchmark`. The cards are
+implemented as the `test-benchmark-sparkline` element.
+
+@demo demo/index.html
+-->
+<dom-module id="test-benchmark-visualizer">
+  <template>
+    <style>
+      :host {
+        @apply(--layout);
```

Comment:
I thiiiiink you don't need `--layout` if you already have `--layout-horizontal`. Also `--layout-flex-horizontal` isn't a thing, you want `--layout-horizontal`. And I don't think you want `--layout-flex`, do you?